### PR TITLE
Fix/ffinput multiple links

### PR DESCRIPTION
**Pull request by fgrunewald**
Fix to issue #243  and some light linting. There are still quite some unused variables and overwritten functions and stuff pylint complains about. But that would require linting all parsers starting with the parser_utilities and then all sub-classes thereof. 

@pckroon Shall I make a separate issue?

### CODE REVIEW

File: vermouth/ffinput.py [hunk 1]
```diff
@@ -799,7 +799,7 @@ def _parse_block_atom(tokens, context):
 def _parse_link_atom(tokens, context, defaults=None, treat_prefix=True):
     if len(tokens) > 2:
         raise IOError('Unexpected column in link atom definition.')
-    elif len(tokens) < 2:
+    if len(tokens) < 2:
```
Comment (pckroon) [inline vermouth/ffinput.py:802]: Why no elif?

File: vermouth/ffinput.py [hunk 2]
```diff
@@ -830,7 +830,7 @@ def _parse_link_atom(tokens, context, defaults=None, treat_prefix=True):
 def _parse_link_attribute(tokens, context, section):
     if len(tokens) > 2:
         raise IOError('Unexpected column in section "{}".'.format(section))
-    elif len(tokens) < 2:
+    if len(tokens) < 2:
```
Comment (pckroon) [inline vermouth/ffinput.py:833]: idem

File: vermouth/ffinput.py [hunk 3]
```diff
@@ -872,7 +872,7 @@ def _parse_meta(tokens, context, context_type, section):
         msg = ('Unexpected column when defining meta attributes for section '
                '"{}" of a {}. {} tokens read instead of 2.')
         raise IOError(msg.format(section, context_type, len(tokens)))
-    elif len(tokens) < 2:
+    if len(tokens) < 2:
```
Comment (pckroon) [inline vermouth/ffinput.py:875]: idem

File: vermouth/ffinput.py [hunk 4]
```diff
@@ -914,7 +914,7 @@ def _parse_patterns(tokens, context, context_type):
 def _parse_variables(tokens, force_field, section):
     if len(tokens) > 2:
         raise IOError('Unexpected column in section "{}".'.format(section))
-    elif len(tokens) < 2:
+    if len(tokens) < 2:
```
Comment (pckroon) [inline vermouth/ffinput.py:917]: idem

File: vermouth/ffinput.py (hunk 1 above)
Comment (fgrunewald) [reply vermouth/ffinput.py:802]: pylint says so ...

**Pull request merged**